### PR TITLE
Core 4249 improve tables 48 k dataset

### PR DESCRIPTION
**Pull request by mariannaparzych**
To run training on modal, you can use a following command (check more details in README.md):
`modal run src/super_gradients/train_from_recipe_modal.py --config-name train_unstructured_yolox.yaml`

### CODE REVIEW

File: src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            enlarge_pixels: size in pixels for bbox enlarging, if None, and enlarge=True, it will be set to 1 px
+            enlarge_classes: list of classes to enlarge bboxes, if None, and enlarge=True, it will be set to all_classes_list
+        """
+        kwargs.pop("subdir", None)
+        kwargs.pop("root", None)
+        if "json_annotation_file" in kwargs:
+            json_file = kwargs["json_annotation_file"]
+            kwargs.pop("json_annotation_file", None)
+
+        # Check whether bboxes should be enlarged by choosed pixel size
```
Comment (amaciaszek-dsai) [inline src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py:61]: choosed -> chosen

File: src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+        x_min -= pixel_size
+        y_min -= pixel_size
+        x_max += pixel_size
+        y_max += pixel_size
+
+        x_min = max(0, x_min)
+        y_min = max(0, y_min)
+
+        x_max = x_max - 1 if x_max >= image_width else x_max
```
Comment (amaciaszek-dsai) [inline src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py:89]: This should be `image_width - 1 if x_max >= image_width else x_max`

or to make it simpler and consistent with what's above:
`min(image_width - 1, x_max)`

File: src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        :return:        Sample, i.e. a dictionary including at least "image" and "target", with enlarged bboxes
+        """
+        if self.enlarge:
+            if self.enlarge_classes is None:
+                self.enlarge_classes = self.classes
+            image_height, image_width, _ = sample["image"].shape
+            for idx, label in enumerate(sample["target"]):
+                class_id = int(label[-1])
+                if self.classes[class_id] in self.enlarge_classes:
+                    sample["target"][idx] = self.change_bbox_coords(image_height, image_width, label, 1)
```
Comment (amaciaszek-dsai) [inline src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py:108]: `1` -> `self.enlarge_pixels`, right?

**Review by amaciaszek-dsai — CHANGES_REQUESTED** [review]

File: src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py (hunk 3 above)
Comment (mariannaparzych) [reply src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py:108]: facepalm :I - fixing

File: src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py (hunk 1 above)
Comment (mariannaparzych) [reply src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py:61]: ok

File: src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py (hunk 2 above)
Comment (mariannaparzych) [reply src/super_gradients/training/datasets/detection_datasets/unstructured_detection.py:89]: ok

**Pull request merged**